.github/workflows/build_and_publish.yaml

```diff
@@ -5,6 +5,10 @@ jobs:
   build_sdist:
     name: Build source distribution
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+      matrix:
+        PYTHON_VERSION: ['3.8', '3.9', '3.10']
```

Comment:
Unfortunately, this doesn't make sense. You just need to build once and publish, not to build 3 times and upload 3. In particular with a package that is architecture independent.

Just hardcode 3.8.

Reply:
Any clue what is breaking with the hardcoded 3.8?

Reply:
This is very odd, I was going to say that you might need to give it as a string, but I see you did that and still was failing. Furthermore, even with subsequent changes, it's still broken and the error is not very helpful.